crates/biome_js_analyze/tests/specs/nursery/useErrorCause/requireCatchParameter.invalid.js

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Notice that this doesn't emit any diagnostics. Add `/* should emit diagnostics */` at the beginning of the file and fix the issue